state/protocol/protocol_state/state/protocol_state.go

```diff
@@ -16,33 +16,65 @@ import (
 // on a per-block and per-epoch basis.
 // It is backed by a storage.ProtocolState and an in-memory protocol.GlobalParams.
 type ProtocolState struct {
-	protocolStateDB storage.ProtocolState
-	globalParams    protocol.GlobalParams
+	protocolStateDB  storage.ProtocolState
```

Comment:
```suggestion
	epochProtocolStateDB  storage.EpochProtocolState
```
Suggesting a possible extension of the rename you've already done, to the storage type and field name.